Improve airgap documentation (#158)
kubewarden/docs: docs/operator-manual/airgap/02-install.md

When the process completes, your current directory will contain downloaded archives. It will be present in the same directory where you executed the command.
```bash
# Print commands but do not execute
airgap pull --list artifacts.json --dry

# Start download
airgap pull --list artifacts.json
```

:::note
Optionally, you can verify the signatures of the [helm charts](../../security/verifying-kubewarden#helm-charts) and [container images](../../security/verifying-kubewarden#container-images)
:::

## Populate private registry

Now you upload archives from previous step into local registry. If you just want to play around you can start local registry with docker.

```bash
# Start local repository (optional)
# docker run -d -p 5000:5000 --restart=always --name registry registry:2

# Push images and policies into local registry
airgap push --list artifacts.json --registry=<REGISTRY.YOURDOMAIN.COM:PORT> --dry --insecure
```

The `sources.yaml` file is needed by kwctl to connect to registries that fall into these categories:
* Authentication is required
* Self signed certificate is being used
* No TLS termination is done

> You can use `--insecure` parameter to skip certificate check, but if you need more controll use [sources.yaml](../../distributing-policies/custom-certificate-authorities.md) file.

Reviewer: ```suggestion
You can use `--insecure` parameter to skip certificate check, but if you need more control use [sources.yaml](../../distributing-policies/custom-certificate-authorities.md) file.
```

Reviewer: I think the note before this sentence on sources.yaml should be moved to follow this sentence stating that the user needs to use it along with kwctl in case they require finer control.

## Install kubewarden

Finally you deploy helm charts. File artifacts.json is the same that was generated in first step.

```bash
airgap install --list artifacts.json --registry <REGISTRY.YOURDOMAIN.COM:PORT>
```

In a few moments you should see kubewarden-controller and policy-server pods running in kubewarden namespace.

We need to configure Policy Server to fetch policies from our private registry. See the [using private registry](../policy-servers/private-registry) section of the docs.

> Alternativelly you can instruct policyserver to ignore certificates by using `--insecure` parameter. This is not recommended.

Reviewer: ```suggestion
Alternatively, you can instruct policyserver to ignore certificates by using the `--insecure` parameter. This is not recommended.
```

> Done. Now we can create Kubewarden policies in our cluster! Policies must be available in your private registry.

Reviewer: ```suggestion
Now we can create Kubewarden policies in our cluster! Policies must be available in your private registry.
```
